sdcm/sct_config.py

```diff
@@ -1007,12 +1007,9 @@ class SCTConfiguration(dict):
         dict(name="cluster_health_check", env="SCT_CLUSTER_HEALTH_CHECK", type=boolean,
              help="When true, start cluster health checker for all nodes"),
 
-        dict(name="validate_partitions", env="SCT_VALIDATE_PARTITIONS", type=boolean,
-             help="when true, log of the partitions before and after the nemesis run is compacted"),
-        dict(name="table_name", env="SCT_TABLE_NAME", type=str,
-             help="table name to check for the validate_partitions check"),
-        dict(name="primary_key_column", env="SCT_PRIMARY_KEY_COLUMN", type=str,
-             help="primary key of the table to check for the validate_partitions check"),
+        dict(name="data_validation", env="SCT_DATA_VALIDATION", type=str,
```

Comment:
shouldn't the type be a dictionary?

Reply:
might be good as well but this way is simpler (less characters) and also looks clearer.

Reply:
did you tried using `SCT_DATA_VALIDATION` to override this one ? I think it might not gonna work as you expect.
I would suggest adding a unittest fort that

but it's can be handled later

Reply:
better later as this PR already contains a big refactor and several bug fixes.